package.json

```
@@ -9,6 +9,8 @@
     "serve": "vue-cli-service serve",
     "test:unit": "vue-cli-service test:unit --coverage",
     "test:e2e": "cypress run",
+    "test:all": "cypress run",
+    "pretest:all": "vue-cli-service test:unit --coverage",
```

You can reference other script calls from within a yarn command and you can separate commands with `&&` to perform multiple. So if I had `yarn foo` and `yarn bar` I could design a third command `yarn foobar` that executes `yarn foo && yarn bar`.
```suggestion
    "test:all": "<what goes here?>"
```

---

Ahhh!! I initially had this but couldn't figure out how to separate the commands. I used a comma, double quotes with a comma, etc. Completely bypassed using a simple &&.  So used the "pre" workaround. Thank you!